site/content/docs/authentication/concepts.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Authentication - Concepts"
+tags: 
+- authentication
+- guide
+type: page
+---
+
+These are the concepts that you will need to understand in order to configure authentication in ZAP.
+
+### Contexts
+
+ZAP [contexts](/docs/desktop/start/features/contexts/) are a way of relating a set of URLs together.
+You can define any contexts you like, but it is expected that a context will correspond to a web application.
+
+Authentication data (including the session handling and user credentials) is associated with a context, so you should have a context which represents the whole of an app protected by authentication.
+
+### Sessions Handling
+
+HTTP is a stateless protocol ([RFC2616](https://www.ietf.org/rfc/rfc2616.txt) section 5), where each request and response pair is independent of other web interactions. [^1]
+
+A session ID or token binds the user authentication credentials (in the form of a user session) to the user HTTP traffic.
+These ID's are typically passed in the HTTP request headers but some apps have been known to pass them via URL parameters or even POST data.
+
+ZAP has to understand how an app manages the user session in order to set the required ID in the HTTP requests that it sends.
+ZAP has built in support for Cookie and HTTP Authentication Session Management but it can also be extended to support any session handling
+method using scripts.
+
+### Authentication Methods
+
+Authentication is the process of verifying that an individual, entity or website is whom it claims to be. Authentication in the context of web applications is commonly performed by submitting a username or ID and one or more items of private information that only a given user should know. [^2]
```

Comment:
```suggestion
Authentication is the process of verifying that an individual, entity, or website is who it claims to be. Authentication in the context of web applications is commonly performed by submitting a username or ID and one or more items of private information that only a given user should know. [^2]
```

Reply:
I'd rather leave this as its a direct quote from the page linked to...